test/dotnet/utils/TestUtils.cs

```diff
@@ -100,6 +100,38 @@ public bool insertBulkCopyWithTransaction(DbConnection bblCnn, DbCommand bblCmd,
 			return true;
 		}
 
+		public bool insertBulkCopyWithKeepIdentity(DbConnection bblCnn, DbCommand bblCmd, String sourceTable, String destinationTable, Logger logger, ref int stCount)
+		{
+			bblCmd.CommandText = "Select * from " + sourceTable;
+			DbDataReader reader = null;
+			DataTable dataTable = new DataTable();
+			try
+			{
+				reader = bblCmd.ExecuteReader();
+				dataTable.Load(reader);
+				reader.Close();
+
+				/* Set KeepIdentity default for this API */
+				SqlBulkCopy bulkCopy = new SqlBulkCopy((SqlConnection)bblCnn, SqlBulkCopyOptions.KeepIdentity, null);
+				bulkCopy.DestinationTableName = destinationTable;
```

Comment:
Why have we introduced a new function? We need to add this option in transinsertbulk and reuse

Reply:
Done